libdnf/goal/Goal.cpp

```diff
@@ -104,6 +105,7 @@ static const std::map<int, const char *> MODULE_PROBLEMS_DICT = {
     {RULE_PKG_NOT_INSTALLABLE_1, _("module %s is disabled")},
     {RULE_PKG_NOT_INSTALLABLE_2, _("module %s does not have a compatible architecture")},
     {RULE_PKG_NOT_INSTALLABLE_3, _("module %s is not installable")},
+    {RULE_PKG_NOT_INSTALLABLE_4, _("module %s is disabled")},
```

Comment:
So I'm looking at this rule thing libsolv has and mildly put I'm getting rather annoyed by it. I don't have the time to spend which it takes to understand it, so I'm not sure what it does as of now. And then this double problem map we have that is indexed by the same keys but with different meanings is another gem...

Rant aside, I'm commenting just to say that this new item you're adding doesn't seem to have a test case (for either of the `PKG_PROBLEMS_DICT` and `MODULE_PROBLEMS_DICT`) in the related ci-dnf-stack PR, which I think it should have, could you add test cases for it?

Reply:
@lukash There is no way how to test it. I have no idea how to trigger this statement with modules. But we have to support it for safety reason. The PR must be merged ASAP. Thanks for understanding.

Reply:
Ok, so just please add a test for the message on line 81, i.e. for:
```
{RULE_PKG_NOT_INSTALLABLE_4, _("package %s is exclude by regular filtering")},
```

Cheers.

Reply:
added